Automattic/wp-calypso
# Add security and trust seals to jetpack checkout

Adds security and trust seals to Jetpack Checkout

## Proposed Changes

* Update the money-back guarantee icon
* Add a Security section below the 7/14 day money-back guarantee

P2: pbNhbs-1c7-p2

## Testing Instructions

1. Checkout this branch or use the calypso live branch
2. Go to `/checkout/jetpack/jetpack_starter_yearly`
3. Ensure you see the security and trust seals below the pay button
![image](https://github.com/Automattic/wp-calypso/assets/65001528/e4cf3807-0eaf-4d18-ab31-1d1087436144)
4. Make sure it scales well to tablet and mobile sizes
5. Go to `/checkout/akismet/ak_personal_yearly:-q-36` and make sure the security and trust seals do not show up here.
![image](https://github.com/Automattic/wp-calypso/assets/65001528/34f6bb65-c019-4175-b033-5679cce238b3)
6. Go to `/checkout/business-bundle` and choose a site. Make sure the seals do not show up here as well
![image](https://github.com/Automattic/wp-calypso/assets/65001528/7fbc19f8-5a29-4a24-be93-1cf3440d1480)

## Pre-merge Checklist
